class.jetpack-idc.php

```diff
@@ -36,13 +36,6 @@ class Jetpack_IDC {
 	 */
 	static $current_screen;
 
-	/**
-	 * The link to the support document used to explain Safe Mode to users
-	 *
-	 * @var string
-	 */
-	const SAFE_MODE_DOC_LINK = 'https://jetpack.com/support/safe-mode';
```

Comment:
@ebinnion do you know if this constant is being used (by partner hosts or otherwise?) - Asking to be sure since [you added a while ago](https://github.com/Automattic/jetpack/commit/a8d8c54060e0d8b5d4a99926d0735b152de6e469), and there are no other uses within Jetpack.

Reply:
Thanks for the ping. I'm not aware of any partners directly using that link.

Reply:
@dereksmart In another situation I opted to leave the constant there with a `deprecated` tag. Maybe it's safer. See https://github.com/Automattic/jetpack/commit/889dbd41a3a7dc9661c0a6cc5b395f0ca0168a13